script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -6,82 +6,85 @@ I stoop down in front of Arcueid.
-no, the scale this time is %{i}much%{/i} greater.
+Arcueid starts to scoop out a bolder as she did before――
+no, it's of far greater scope this time――
+she scoops out the very ground, rubble included, changing the terrain in the process.
 }
 [sha:7a8ba45340de7d451b6e194b8e00a021e27459df]{
 -- Page 660, Offset 10425.
 -- それは爆発を思わせる、大地そのものの隆起だった。
-The earth itself erupted. You'd be forgiven for thinking a bomb had gone off.
+The earth itself erupts. You'd be forgiven for thinking a bomb had gone off.
 }
 [sha:31a2d2135a66b7f81bd20088656aea4a539bd2cf]{
 -- Page 660, Offset 10426.
 -- 木っ端微塵の力技。ヴローヴの冷気が吹雪なら、アルクェイドの所行は大地の津波だ。
-The power of an overwhelming number of fragments.
+//木っ端微塵の力技 is poetic, and shouldn't be taken too literally, as with most of the coming lines.
+It is a feat of strength that renders all before it to dust.
  If Vlov's cold air is like a blizzard,
  then what Arcueid did is like an earth tsunami.
 }
 [sha:afef4e00d09328d323a22a176d37427de0f746d6]{
 -- Page 661, Offset 10427.
 -- 飛翔する氷の槍など、この壁の如き土砂の前では飛沫にすぎない。
-A flying ice spear is nothing more than a droplet of water before this wall of earth and sand.
+Vlov's paltry soaring spears of ice may as well be as droplets of rain before this wall of earth and sand.
 }
 [sha:c73702798442718915e7264a11a3280ca85d4e3a]{
 -- Page 661, Offset 10428.
 -- 高さ50メートル、7000トンに相当する質量が、ヴローヴを飲み込まんと押し寄せる。
 The mass, 50 meters tall and weighing 7000 tons,
- bears down on Vlov as if to swallow him whole.
+ bears down on Vlov as to swallow him whole.
 }
 [sha:9dd9d12d247e76a4030a2d9b9b1d05a8285ea39c]{
 -- Page 662, Offset 10429.
 -- されど、地を返す天変地異に<騎|ヤ><士|ツ>は動じず。
 And yet,
- even in the face of this natural disaster that has turned the earth's surface inside out,
- the knight doesn't budge.
+ even in the face of this earth-shattering cataclysm,
+ does the knight not budge.
 }
 [sha:e4f0fe3300f8ddb1ce8792301d79f29142589838]{
 -- Page 662, Offset 10430.
 -- 土砂の大波を、鋼鉄の槍の一刺しで貫き、押し返す。
-The billow of earth is driven back by a single thrust from his steel lance.
+The deluge of soil is driven back by a single thrust from his steel lance.
 }
 [sha:8cc99c482015cce5197240150ee92acb3d33a1ac]{
 -- Page 663, Offset 10431.
 -- 瓦礫の土砂降りの中、ヴローヴは視認したハズだ。
-Vlov must have seen through the downpour of debris.
+Vlov should've already confirmed what happened as he was subsumed by the downpour of debris.
 }
 [sha:a73a62161a1d455fabdcaced3fff75a3791ba0bc]{
 -- Page 663, Offset 10432.
 -- 最後の力で岩盤を両手でひっくり返し、その消耗から膝をついたアルクェイドと、おそらく、この土砂を目くらましにして接近する“脅威”が、どこにも見当たらない事を。
-He must have seen Arcueid fall to her knees after using the last of her strength to upend that bedrock,
- and concluded that the 'threat' supposedly lurking behind this smokescreen was nowhere to be found.
+Both that Arcueid fell to her knees after she used up the last of her strength in upending that bedrock,
+ and that the 'threat' who, in all likelihood, was approaching under the smokescreen of earth is now nowhere to be found.
```

Comment:
This and the previous line is a set, so I'll just do both here.
I think the active aspect here it unneeded, so adjusting
>Even through the downpour of debris, Vlov could surely still see what happened.

He then goes on to describe what happened:
>First, that Arcueid fell to her knees after using the last of her strength to upend that bedrock. 
>And then, that the threat――who in all likelihood was approaching him by using the soil as a smokescreen――is now nowhere to be found.

The last one is actually then what we see from Vlov's perspective as it plays back what happened from his view:
>The man's eyes widen as the vast expanse in front of him comes into view.'
>―――Gone. Not a trace to be found.

So  どこにも見当たらない事を。he has already 'found out' that for some reason, Shiki is gone. He thought he was coming using the soil as a smokescreen, but he's not, he's gone. It then leaves that on sort of a cliffhanger, until the flashback catches up.

Reply:
If that's the intent (he is seeing that the person he _assumed_ was using the smokescreen _is_ gone for other reasons), then the middle bit needs to be changed - 
> First, that Arcueid fell to her knees after using the last of her strength to upend that bedrock.
And second, that the threat――who he'd assumed was approaching him using the soil as a smokescreen――is now nowhere to be found.

but I don't think that's yet true here? He doesn't know that Shiki _isn't_ behind the wall until after he's already stabbed it, and at the point in time this line is talking about we're still **before** that has happened. Otherwise why would he even be thinking about holding back when he stabs it?

Reply:
He's already stabbed it here:
Previous line:

> The deluge of soil is driven back by a single thrust from his steel lance.

The Vlov part then goes over it from his perspective, _again_.

Reply:
The line comes after, but it's talking about something that happened before

Reply:
- He strikes the wall
- Shiki then says ok, when Arcueid threw up the wall, he must have still been able to see through it enough to ascertain that Arc has fallen down and that he can no longer spot Shiki. He then deduced that Shiki is using the wall to advance unseen, and so destroyed it when it got close
- Flashbacks happen
- Vlov strikes the wall for real this time from his own perspecive
- _Only then_ does he see that Shiki _isn't_ behind it

Reply:
> With the debris scattered by his strike, he must be able to see it now. 
> First, that Arcueid fell to her knees after using the last of her strength to upend that bedrock.
> And second, that the threat――who he'd assumed was approaching him using the soil as a smokescreen――is now nowhere to be found.